checkov/terraform/checks/resource/ncp/AccessControlGroupOutboundRule.py

```diff
@@ -0,0 +1,25 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class AccessControlGroupOutboundRule(BaseResourceCheck):
+    def __init__(self):
+        name = "An outbound security group rule allows traffic to /0"
```

Comment:
```suggestion
        name = "Ensure no security group rules allow outbound traffic to 0.0.0.0/0"
```
typically we start our check names with `Ensure ...`